tests/scalers/cassandra.test.ts

```diff
@@ -52,15 +57,17 @@ test.before(t => {
     sh.exec(`kubectl exec ${cassandraClientPod} -n ${cassandraNamespace} -- bash cqlsh -u ${cassandraUsername} -p ${cassandraPassword} cassandra.${cassandraNamespace} --execute="${createTableCQL}"`)
 
     // deploy nginx, scaledobject etc.
+    console.log("deploy nginx, scaledobject etc.")
     const nginxTmpFile = tmp.fileSync()
     fs.writeFileSync(nginxTmpFile.name, nginxDeployYaml)
 
     t.is(0, sh.exec(`kubectl apply --namespace ${cassandraNamespace} -f ${nginxTmpFile.name}`).code, 'creating nginx deployment should work.')
     // wait for nginx to load
+    console.log("wait for nginx to load")
     let nginxReadyReplicaCount = '0'
      for (let i = 0; i < 30; i++) {
         nginxReadyReplicaCount = sh.exec(`kubectl get deploy/${nginxDeploymentName} -n ${cassandraNamespace} -o jsonpath='{.status.readyReplicas}'`).stdout
-        if (nginxReadyReplicaCount != '1') {
+        if (nginxReadyReplicaCount != '') {
```

Comment:
Don't follow this change?

Reply:
It starts with 0 replica at first, not 1, so this check was wrong

Reply:
In that case, should it be different from 0 instead of empty?

Reply:
I'm not sure if I got your question right, so do you mean it should be like this:
`if (nginxReadyReplicaCount != '0') {`

When I stdout the nginxReadyReplicaCount at that stage, it prints empty, that's why I changed it this way. However, I don't think that checking a deployment with 0 replica at the beginning whether it is empty or zero makes sense. This control is needless, I guess. WDYT?

Reply:
It depends on the test itself. I mean, if the test check that the scaler scales from 0 to X, IMHO it's a good idea checking that the initial state is really 0 instead of any other initial state that could be produced by a side effect. 
In general (IMO again), always it's good idea during the tests to assert the initial state. 
Any opinion here @zroubalik ?

BTW, you could use the existing helper to check the current amount of instances instead of querying directly. It is easier to read and maintain. [This is a usage example](https://github.com/kedacore/keda/blob/main/tests/scalers/rabbitmq-queue-http.test.ts#L36)